css/styles.css

```diff
@@ -183,13 +183,20 @@
 
 .greenText {
   color: #92dcdc;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` then assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.